### PR TITLE
remove superfluous `require time` from gigasecond test

### PR DESCRIPTION
**Pull request by hilary**

### CODE REVIEW

**Comment by kotp** [comment]
Thank you @hilary.

This test file is generated.  I think the change belongs in the `exercises/gigasecond/example.tt` file, which will generate the test file.

**Comment by hilary** [comment]
Doh! Learning my way around ... and here I thought I had read through the docs 😊

**Comment by kotp** [comment]
Thank you @hilary it is all good...

**Comment by kotp** [comment]
I missed it, doh!  Time was not meant to be removed from here, as Time is being used as a convenience in the tests, not a requirement for the solution.  This makes the solution a bit less flexible.

File: exercises/gigasecond/example.tt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 require 'minitest/autorun'
-require 'time'
```
Comment (kotp) [inline exercises/gigasecond/example.tt]: This should not have been removed.

**Pull request merged**